src/services/ui/src/components/Modal/ConfirmationModal.tsx

```diff
@@ -12,6 +13,7 @@ export type ConfirmationModalProps = {
   open: boolean;
   description?: React.ReactNode;
   body?: React.ReactNode;
+  reverseButtonPlacement?: boolean;
```

Comment:
What's the naming convention for boolean?

Reply:
I prefer to have them be questions like `isReverseButtonPlacement`. No hard and fast rule though.